docs/codespell.txt

```diff
@@ -0,0 +1,4 @@
+requre
+noveau
+checkt
+hax
```

Comment:
I'd replace this with `TODO`, hardly worthy of an exception.

Reply:
:D I likes the word hax, but sure